### PR TITLE
feat: CSV track param reader

### PR DESCRIPTION
**Pull request by andiwand**
adds a CSV track parameter reader to the examples

### CODE REVIEW

**Review by AJPfleger — COMMENTED** [review]
Just marginally a part of this PR, but is there I reason, why we have the header file `Examples/Io/Csv/src/CsvOutputData.hpp` in the `src`-directory instead of `include`?

Also I think we need to update the (c)-dates in the files, we have modified.

**Comment by andiwand** [comment]
> is there I reason, why we have the header file `Examples/Io/Csv/src/CsvOutputData.hpp` in the `src`-directory instead of `include`?

I can only guess but it looks like the csv structs are not used anywhere outside and are therefore more like an implementation detail.

**Comment by AJPfleger** [comment]
Ups, I see, I was very vague. With the year-update I meant `2019-2022`

File: Examples/Io/Csv/src/CsvOutputData.hpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2019 CERN for the benefit of the Acts project
+// Copyright (C) 2022 CERN for the benefit of the Acts project
```
Comment (andiwand) [inline Examples/Io/Csv/src/CsvOutputData.hpp:3]:
```suggestion
// Copyright (C) 2019-2022 CERN for the benefit of the Acts project
```

**Comment by AJPfleger** [comment]
> > ... is there I reason, why we have the header file `Examples/Io/Csv/src/CsvOutputData.hpp` in the `src`-directory instead of `include`?
> 
> I can only guess but it looks like the csv structs are not used anywhere outside and are therefore more like an implementation detail.

@paulgessinger ?

**Review by asalzburger — APPROVED** [review]
Looks good,  let's merge.

**Pull request merged**